Datadog/Example/Debugging/DebugWebviewViewController.swift

```diff
@@ -25,21 +25,24 @@ class DebugWebviewViewController: UIViewController {
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         if useNativeRUMSession {
-            Global.rum.startView(viewController: self)
+            RUMMonitor.shared()
```

Comment:
non blocking

instance member would be nicer

Reply:
I will keep it as is for the same reason as [here](https://github.com/DataDog/dd-sdk-ios/pull/1236#discussion_r1189433437) 👍